product_configurator/models/product.py

```diff
@@ -347,44 +351,35 @@ def _check_config_line_domain(self):
             domain_value_ids = domain_id.domain_line_ids.mapped("value_ids")
             invalid_value_ids = domain_value_ids - tmpl_value_ids
             invalid_attribute_ids = domain_attr_ids - tmpl_attribute_ids
-            if not invalid_value_ids and not invalid_value_ids:
+            if not invalid_attribute_ids and not invalid_value_ids:
                 continue
             if not error_message:
                 error_message = _(
                     "Following Attribute/Value from restriction "
                     "are not present in template attributes/values. "
                     "Please make sure you are adding right restriction"
                 )
-            error_message += _("\nRestriction: %s") % (domain_id.name)
+            error_message += _("\nRestriction: %s", domain_id.name)
             error_message += (
                 invalid_attribute_ids
-                and _("\nAttribute/s: %s")
-                % (", ".join(invalid_attribute_ids.mapped("name")))
+                and _(
+                    "\nAttribute/s: %s", ", ".join(invalid_attribute_ids.mapped("name"))
+                )
```

Comment:
This will generate an incorrect translation string.

Reply:
Sorry but I don't see how, could you please make an example?

Reply:
I misread, it's OK